src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -353,7 +395,7 @@ def _local_file_resolver(self) -> BaseFileResolver | None:
     def build_notebook_dependency_graph(self, path: Path, session_state: CurrentSessionState) -> MaybeGraph:
         """Builds a dependency graph starting from a notebook. This method is mainly intended for testing purposes.
         In case of problems, the paths in the problems will be relative to the starting path lookup."""
-        maybe = self._notebook_resolver.resolve_notebook(self._path_lookup, path)
+        maybe = self._notebook_resolver.resolve_notebook(self._path_lookup, path, False)
```

Comment:
Why is the `inherit_context` `False` here? And could you make this a keyword argument to improve readability?

Reply:
Same question as in `jobs.py`

Reply:
done